deploy/operator/gather.sh

```diff
@@ -157,6 +157,11 @@ function gather_hypershift_data() {
   hypershift_dir="${LOGS_DEST}/hypershift"
   mkdir -p "${hypershift_dir}"
 
+  oc get po -n hypershift -o yaml > ${hypershift_dir}/hypershift_pods.yaml
+  oc get po -n hypershift -o custom-columns=:.metadata.name --no-headers | while read hypershift_pod_name ; do
```

Comment:
Already collected [here](https://github.com/openshift/assisted-service/pull/5344/files#diff-9ff8483d183f18ac66f280b1002c37eba28d5d3be3078a02579738696e4ca3fcR143)
Consider collecting the yaml there or moving the log collection here

Reply:
Thanks for pointing this out! Moved yaml collection there